images/linux/scripts/base/limits.sh

```diff
@@ -5,3 +5,6 @@ echo '* hard nofile 65536' >> /etc/security/limits.conf
 echo 'session required pam_limits.so' >> /etc/pam.d/common-session
 echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive
 echo 'DefaultLimitNOFILE=65536' >> /etc/systemd/system.conf
+
+# Double stack size from default 8192KB
+ulimit -s 16384
```

Comment:
I think we should use `/etc/security/limits.conf` to apply permanent changes.

Reply:
Sorry, yes definitely!

Reply:
LGTM.

![image](https://user-images.githubusercontent.com/47745270/116198309-794f8e80-a73e-11eb-9264-0dbcc518c0a1.png)